h2o-genmodel/src/main/java/hex/genmodel/ModelMojoReader.java

```diff
@@ -252,6 +256,7 @@ else if (line.equals("[columns]")) {
   }
 
   private String[][] parseModelDomains(int n_columns) throws IOException {
+    final boolean multilineCategoricals = existsInKV("multiline_categoricals") && (boolean) readkv("multiline_categoricals"); // The key might not exist in older MOJOs
```

Comment:
This can cause NPE if the key is present but the corresponding value is null.

Safe way to do the same thing:

    Boolean.TRUE.equals(readkv("multiline_categoricals"))

Reply:
Thanks for catching it for me. Deleting existsInKVMethod (unused now).